Corona-Warn-App/src/main/java/de/rki/coronawarnapp/srs/core/repository/SrsSubmissionRepository.kt

```diff
@@ -115,7 +116,9 @@ class SrsSubmissionRepository @Inject constructor(
             srsOtp
         } else {
             Timber.tag(TAG).d("SRS otp is not valid -> new otp generated")
-            SrsOtp()
+            SrsOtp().also {
+                OtpCensor.otp = it
+            }
```

Comment:
you already doing this above , it should cover this case too , right?

Reply:
yes, removed